tests/test_integration_http.py

```diff
@@ -33,19 +33,17 @@ def test_capture_http(cmake, httpserver):
     httpserver.expect_oneshot_request(
         "/api/123456/envelope/", headers={"x-sentry-auth": auth_header},
     ).respond_with_data("OK")
+    env = dict(os.environ, SENTRY_DSN=make_dsn(httpserver), SENTRY_RELEASE="🤮🚀")
 
-    with httpserver.wait(raise_assertions=True, stop_on_nohandler=True) as waiting:
```

Comment:
Is removing the waiting here safe?

Reply:
Yes, I actually had problems with the wait before. When the subprocess exits, things are guaranteed to be sent.